cli/blade-jar-tests.gradle

```diff
@@ -0,0 +1,101 @@
+allprojects {
+	tasks.withType(JavaExec) {
+		dependsOn jar
+	}
+}
+
+task testCreateCommand(type: JavaExec)
+task testGwCommand(type: JavaExec)
+task testHelpCommand(type: JavaExec)
+task testInitCommand(type: JavaExec)
+task testSamplesCommand(type: JavaExec)
+task testVersionCommand(type: JavaExec)
+
+check {
+	dependsOn tasks.withType(JavaExec)
+}
+
+def errorStream = new ByteArrayOutputStream()
+def standardStream = new ByteArrayOutputStream()
+
+testCreateCommand {
+	args = [jar.archivePath, 'create', '-l']
+	errorOutput = errorStream
+	main = '-jar'
+	standardOutput = standardStream
+
+	doLast{
+		assert standardStream.toString().contains("service-builder")
+		assert !errorStream.toString().toLowerCase().contains("error")
```

Comment:
@gamerson, javaexec doesnt have result, I'd have to switch to exec to assert the exit codes